config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-helm-test.yaml

```diff
@@ -0,0 +1,37 @@
+# sigs.k8s.io/descheduler presubmits
+presubmits:
+  kubernetes-sigs/descheduler:
+  - name: pull-descheduler-verify-helm-test
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: pull-descheduler-verify-helm-test
+    decorate: true
+    path_alias: sigs.k8s.io/descheduler
+    branches:
+    # The script this job runs is not in all branches.
+    - ^master$
+    always_run: true
+    spec:
+      containers:
+      - image: golang:1.15.5
+        command:
+        - make
+        args:
+        - verify
```

Comment:
I don't think you actually need this? we already run `make verify` in the presubmits: https://github.com/kubernetes/test-infra/blob/69cad13/config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-master.yaml#L2-L20

in fact you could probably just add the `make test-helm` stanza below to that file instead of creating a new one

Reply:
Got it, like you mentioned it'll be better if I add it in the pre-submits file. I'll update the code.

Reply:
Updated the code with suggested changes